src/include/update/rootpart.rb

```diff
@@ -63,12 +63,7 @@ def CanBeLinuxRootFS(partition_fs)
         return false
       end
 
-      # possible_root_fs contains list of supported FSs
-      # storage-ng
-      # FIXME: this kind of checks should be responsibility of the upcoming
-      # Y2Storage::FilesystemType (to be added as part of the
-      # "yast-storage-ng as a libstorage wrapper" change)
-      possible_root_fs = [:ext2, :ext3, :ext4, :btrfs, :reiser, :xfs]
+      possible_root_fs = Y2Storage::Filesystems::Type.root_filesystems().map { |x| x.to_sym }
       possible_root_fs.include?(partition_fs)
```

Comment:
NP: just for the record, this could also be
```ruby
possible_root_fs = Y2Storage::Filesystems::Type.root_filesystems
possible_root_fs.any? { |fs| fs.is?(partition_fs) }
```